packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -211,12 +211,13 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
         </button>
       );
     } else if (item.href) {
+      // Allow the disabled property on anchor elements for commandbar
       command = (
         <a
-          { ...getNativeProps(item, anchorProperties) }
+          { ...getNativeProps(item, anchorProperties.concat(['disabled'])) }
```

Comment:
if we want disabled to be a valid native prop, which it seems like it should be, then it should get added to properties.ts

Reply:
I'm not sure that we want the disabled attribute to be added to anchor and div elements. The disabled attribute is not a native HTML attribute for those tags. I was intending for the attribute to be valid for command bar items only. I've added a comment to clarify.  
https://www.w3schools.com/tags/att_disabled.asp